examples/tree_2d_dgsem/elixir_shallowwater_three_mound_dam_break.jl

```diff
@@ -0,0 +1,113 @@
+
```

Comment:
See comment above for `StructuredMesh` version of this elixir. This one should be removed because this test case requires a rectangular domain.